api/src/main/java/org/apache/cloudstack/api/command/user/vm/DeployVMCmd.java

```diff
@@ -256,12 +281,35 @@ public Long getDomainId() {
                 }
             }
         }
+        if(getBootType() != null){ // export to get
+            if(getBootType() == ApiConstants.BootType.UEFI) {
```

Comment:
@pavanaravapalli cc @DaanHoogland @andrijapanicsb 
There is no null check, write it as `someEnum.equals(someVariable or Method())` - this will never throw NPE.